test/test_ops.py

```diff
@@ -59,6 +59,27 @@ def test_supported_dtypes(self, device, dtype, op):
             self.assertTrue(sample_input.dtype == dtype)
             self.assertTrue(sample_input.device.type == self.device_type)
 
+    # Verifies that backward for each unsupported floating or complex dtype
+    #   throw a runtime error.
+    @onlyOnCPUAndCUDA
+    @ops(op_db, dtypes=OpDTypes.unsupported_backward,
+         allowed_dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16))
+    def test_unsupported_backward(self, device, dtype, op):
+        if not op.supports_autograd:
+            self.skipTest("Skipped! Autograd not supported.")
+        try:
+            samples = op.sample_inputs(device, dtype, requires_grad=True)
+        except RuntimeError as e:
+            self.skipTest(f"Skipped! unable to generate sample. {e}")
+
+        for sample in samples:
```

Comment:
Let's skip if the output is not a tensor, because that's really a limitation of the test

```
for sample in samples:
  result = op(sample.input, *sample.args, **sample.kwargs)

  # TODO: handle non-tensor outputs
  if not isinstance(result, torch.Tensor):
    self.skipTest(f"Skipped! Test does not handle non-tensor outputs")

  with self.assertRaises(RuntimeError):
    result.sum().backward()
```